example/package.json

```diff
@@ -4,42 +4,50 @@
   "version": "0.1.0",
   "homepage": "./",
   "scripts": {
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
-    "format": "prettier-package-json --write && prettier --loglevel warn '**/*.{ts,tsx,js,json,md}' '!build/**' --write",
+    "prebuild": "tsc",
+    "build": "vite build",
+    "dev": "vite",
+    "format": "prettier-package-json --write && prettier --loglevel warn '**/*.{ts,tsx,js,.cjs,json,md}' '!build/**' --write",
     "lint": "yarn lint:js && yarn lint:md",
-    "lint:js": "eslint src/ --ext js,ts,tsx --max-warnings 0",
+    "lint:js": "eslint . --ext js,ts,tsx",
     "lint:md": "markdownlint README.md",
     "prepare": "yarn build",
-    "start": "react-scripts start",
-    "test:ci": "yarn format --check && yarn lint"
+    "start": "yarn dev",
+    "test:ci": "yarn format --check && yarn lint",
+    "preview": "vite preview"
```

Comment:
Can we stick with standard Garden scripting and parity with previous?
- modify to `"start": "vite --open"`
- no `dev`
- no `preview` (similar to `website` where rare production run can be executed as needed)

Reply:
Sure, I've updated in [fb65a8a](https://github.com/zendeskgarden/tailwindcss/pull/150/commits/fb65a8a3b0c4ac835cdea1fd6902b5269c2ddf29).